plutus-metatheory/src/Main.lagda

```diff
@@ -261,22 +261,24 @@ reportError (runtimeError gasError)         = "gasError"
 reportError (runtimeError userError)        = "userError"
 reportError (runtimeError runtimeTypeError) = "runtimeTypeError"
 
+checkError : ∀{A} → ∅ ⊢ A → Either ERROR (∅ ⊢ A )
```

Comment:
For Haskell code we encourage Haddock for all functions and modules (see style guide). I'm not sure for Agda. But I think it will be helpful to gradually add more documentation to the Agda code base. We are working on decreasing key person risk in our team.

Reply:
I was mostly following the style of current code,  but you are right that more documentation should be added.

Reply:
I have added some comments.

Reply:
Thanks! It's helpful!